src/components/Garden/Feed/ProposalsList.tsx

```diff
@@ -58,6 +65,14 @@ function ProposalsList({
     }
   }, [proposals.length]) //eslint-disable-line
 
+  // TODO: Need to be discussed
+  const Card = (proposal: ProposalType) =>
+    proposal.metadata === 'Abstain proposal' ? (
+      <AbstainCard proposal={proposal} />
+    ) : (
+      <ProposalCard proposal={proposal} />
+    )
+
```

Comment:
That yet need be discussed?

Reply:
It's not. I need to talk with Gabi is this is the correct way.